index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+        rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <!-- fontawesome -->
+    <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./assets/css/main.css">
+    <link rel="stylesheet" href="./assets/css/common.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <title>Tech-Talk Conference</title>
+</head>
+<body>
+    <nav id = "desk-most-top">
+        <ul class="most-top">
+            <li>
+                <a href="#" class="icon">
+                    <i class="fab fa-facebook fa-fw"></i>
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <i class="fab fa-twitter fa-fw"></i>
+                </a>
+            </li>
+            <li class="icon">English</li>
+            <li>
+                <a href="#" class="icon">My Page</a>
+            </li>
+            <li>
+                <a href="#" class="icon">Logout</a>
+            </li>
+        </ul>
+
+        <div id="main-nav">
+            <div>
+                <a href="#"  class="nav-logo"><img src="https://www.transparentpng.com/thumb/business/picture-business-15.png" alt="Icon Image"></a>
+            </div>
+            <ul class="right-nav">
+                <li class="right-nav-link">
+                    <a href="#">Home</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="./page/about.html">About</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="#main-program">Program</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="#feature-speaker">Speaker</a>
+                </li>
+                <li class="right-nav-link">
+                    <a href="#partner">Partner</a>
+                </li>
+                <li class="right-nav-link most-right-link">
+                    <a href="#">Tech Campion</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <section class="mobile-nav">
+        <h1></h1>
```

Comment:
- [ ] I will suggest you remove this `h1 tag` since you are not using it.

Reply:
Hi @enaburekhan 

You mentioned great point but linter will rise an error if didn't use it. That is why I left it blank. For sure We will find sth to write on it in the future. 🌝

Reply:
Hi @bushmusi, following semantic HTML, it is not recommended to have more than one `h1 tag` on an HTML page. So you need to find a way to fix the error. Leaving it empty wouldn't be the best solution.

Reply:
Hi @enaburekhan 

You are right 💯 . I have removed that and `css` linter passed successfully